### PR TITLE
Added document with supported platforms [9368]

### PR DESCRIPTION
**Pull request by imontesino**
Signed-off-by: imontesino <monte.igna@gmail.com>

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Support
+
+This document reflects the level of support offered by *eprosima Fast DDS* on different platforms as per the following
+definitions:
+
+Tier 1
```
Comment (MiguelCompany) [inline SUPPORT.md:6]: Please add tiers as headers

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Tier 1 platforms are subjected to our unit test suite and other testing tools on a frequent basis including continuous
+integration jobs, nightly jobs, packaging jobs, and performance testing.
+Errors or bugs discovered in these platforms are prioritized for correction by the development team.
+Significant errors discovered in Tier 1 platforms can impact release dates and we strive to resolve all known high
+priority errors in Tier 1 platforms prior to new version releases.
+
+Tier 2
+
+Tier 2 platforms are subject to periodic CI testing which runs both builds and tests with publicly accessible results.
+The CI is expected to be run at least within a week of relevant changes for the current state of the ROS distribution.
```
Comment (MiguelCompany) [inline SUPPORT.md:17]: `ROS distribution` ?

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+The development team does not run the unit test suite or perform any other tests on platforms in Tier 3.
+Community members may provide assistance with these platforms.
+
+## Platforms
+
+|Architecture|Ubuntu Focal (20.04)|MacOS Mojave (10.14)|Windows 10 (VS2019)|Debian Buster (10)|
+|------------|--------------------|--------------------|-------------------|------------------|
+|amd64       |Tier 1 [a][s]       |Tier 1 [s]          |Tier 1 [a][s]      |Tier 3 [s]        |
+|arm64       |Tier 1 [a][s]       |                    |                   |Tier 3 [s]        |
+|arm32       |Tier 3 [s]          |                    |Tier 2 [s]         |Tier 3 [s]        |
```
Comment (MiguelCompany) [inline SUPPORT.md:38]: Are we building for `Windows 10`  on `arm32` ?

**Review by MiguelCompany — CHANGES_REQUESTED** [review]
I think we should change the name of the file to `PLATFORMS.md` or `PLATFORM_SUPPORT.md`

File: SUPPORT.md (hunk 3 above)
Comment (imontesino) [reply SUPPORT.md:39]: No, it is a mistake. We are building for amd32.

**Review by MiguelCompany — APPROVED** [review]

**Pull request merged**